### PR TITLE
GH-1429 Add WebSockets and bump mininum version numbers

### PR DESCRIPTION
**Pull request by IAmThePan**
Added WebSockets bug checking for WebSockets!
Bumped minimum browser versions for websocket support.

Need to test this heavily on Edge.

### CODE REVIEW

**Comment by zarembsky** [comment]
ws, wss settings in background.js should be gated from Edge

**Comment by IAmThePan** [comment]
Fixed by not listening to WebSockets on Edge.  @zarembsky please review again.

**Comment by zarembsky** [comment]
Looks good.

File: src/background.js [hunk 1]
```diff
@@ -1419,9 +1419,12 @@ function initializePopup() {
...
 function addCommonGhosteryAndAntitrackingListeners() {
-	chrome.webRequest.onBeforeRequest.addListener(onBeforeRequest, {
-		urls: ['http://*/*', 'https://*/*']
-	}, ['blocking']);
+	// Edge doesn't support WebSockets
+	const urls = ['http://*/*', 'https://*/*'];
+	if (!IS_EDGE) {
+		urls.push('ws://*/*', 'wss://*/*');
+	}
+	chrome.webRequest.onBeforeRequest.addListener(onBeforeRequest, { urls }, ['blocking']);
```
Comment (jsignanini) [inline src/background.js]: @christophertino is there a reason why we don't also add these to the `onHeadersReceived` listener?
